src/ServiceControl.AcceptanceTests/Recoverability/When_a_message_is_retried_with_a_replyTo_header.cs

```diff
@@ -16,7 +16,12 @@
     public class When_a_message_is_retried_with_a_replyTo_header : AcceptanceTest
     {
         // TODO: Add these transports back if/when then are updated to match this behavior
-        [Test, IgnoreTransports("AzureServiceBus", "AzureStorageQueues", "RabbitMq")]
+        [Test, IgnoreTransports(
+             "AzureServiceBus - Endpoint Topology", 
+             "AzureServiceBus - Forwarding Topology", 
```

Comment:
I would assume these ignores have been written with the topology at that time in mind. Under those circumstances, I would suggest we remove  "AzureServiceBus - Forwarding Topology" and "RabbitMq - Direct Routing Topology" because those have been introduced recently. Or should we say if something did not work under Endpoint Topology it will not work under Forwarding either?